packages/wonder-blocks-dropdown/components/single-select-menu.test.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import {mount} from "enzyme";
 
 import SelectBox from "./select-box";
-import SelectItem from "./select-item";
+// import SelectItem from "./select-item";
 import SingleSelectMenu from "./single-select-menu.js";
 
 const keyCodes = {
```

Comment:
I can't comment on these lines, but you'll want to move your shared setup into a `beforeEach` block to isolate tests better.